rest/taskana-rest-spring-example-boot/pom.xml

```diff
@@ -95,8 +95,25 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
```

Comment:
Please use the aggrigator junit-jupiter

Reply:
removed test classes and test dependencies